base/src/main/java/edu/artic/base/utils/AppBarHelper.kt

```diff
@@ -0,0 +1,25 @@
+package edu.artic.base.utils
+
+import android.support.design.widget.AppBarLayout
+import android.widget.TextView
+
+class AppBarHelper {
+    companion object {
```

Comment:
This reminds me of the java way of doing it. why not make an extension on `AppBarLayout` as a top level method? would be easier to discover.